sr_demos/scripts/sr_demos/open.py

```diff
@@ -74,11 +74,5 @@
         }
     ]
 
-    if hand_name == "two_hands":
-        lh_commander_instance = SrHandCommander("left_hand")
-        rh_commander_instance = SrHandCommander("right_hand")
-        lh_commander_instance.run_named_trajectory(trajectory)
-        rh_commander_instance.run_named_trajectory(trajectory)
-    else:
-        hand_commander_instance = SrHandCommander(name=hand_name)
-        hand_commander_instance.run_named_trajectory(trajectory)
+    commander_instance = SrHandCommander(hand_name)
+    commander_instance.run_named_trajectory(trajectory)
```

Comment:
Same on this file.

Reply:
sorry about that. fixed in 4b2d32fdb479c15c8e3e04278ba59fd405ef4f04